### PR TITLE
PWX-25488: add option to disable some objects from controller list-wa…

### PR DESCRIPTION
**Pull request by haibinxie**
…tch cache to reduce memory usage

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: cmd/operator/operator.go [hunk 1]
```diff
@@ -247,11 +257,57 @@ func run(c *cli.Context) {
...
+			objs = append(objs, &rbacv1.ClusterRole{})
+		case "ClusterRoleBinding":
+			objs = append(objs, &rbacv1.ClusterRoleBinding{})
+		case "Role":
+			objs = append(objs, &rbacv1.Role{})
+		case "RoleBinding":
+			objs = append(objs, &rbacv1.RoleBinding{})
+		case "StorageClass":
+			objs = append(objs, &storagev1.StorageClass{})
+		}
```
Comment (pure-jliao) [inline cmd/operator/operator.go:298]: nit: better to add else, if the type is not supported we should print a log

File: cmd/operator/operator.go [hunk 2]
```diff
@@ -247,11 +257,57 @@ func run(c *cli.Context) {
...
+}
+
 func createManager(c *cli.Context, config *rest.Config) (manager.Manager, error) {
 	syncPeriod := defaultResyncPeriod
 	managerOpts := manager.Options{
-		SyncPeriod:         &syncPeriod,
-		MetricsBindAddress: fmt.Sprintf("0.0.0.0:%d", c.Int(flagMetricsPort)),
+		SyncPeriod:            &syncPeriod,
+		MetricsBindAddress:    fmt.Sprintf("0.0.0.0:%d", c.Int(flagMetricsPort)),
+		ClientDisableCacheFor: getObjects(c.String(flagDisableCacheFor)),
```
Comment (pure-jliao) [inline cmd/operator/operator.go:310]: if the input is empty, does it affect anything?

**Review by pure-jliao — COMMENTED** [review]
LGTM, left some comments.

File: cmd/operator/operator.go (hunk 2 above)
Comment (haibinxie) [reply cmd/operator/operator.go:310]: it does not, default is empty list.

File: cmd/operator/operator.go (hunk 1 above)
Comment (haibinxie) [reply cmd/operator/operator.go:298]: sounds good

**Review by pure-jliao — APPROVED** [review]

**Pull request merged**